cmd/alertmanager/main.go

```diff
@@ -129,6 +129,10 @@ func newMarkerMetrics(marker types.Marker) {
 const defaultClusterAddr = "0.0.0.0:9094"
 
 func main() {
+	os.Exit(run())
+}
+
+func run() int {
```

Comment:
this is so close to fixing another issue, the integration tests relying on a locally built binary.. if you have time, could you expand this to take all the command line flags as arguments (or options/config struct), and then use this function in the integration tests? a separate PR is cool

Reply:
Yep